### PR TITLE
Add support for domain-mapping plugin from WPMUDEV

### PR DESCRIPTION
**Pull request by dgwatkins**
Comes from compsupp-2737

### CODE REVIEW

File: changelog.md
Comment (strategio) [inline changelog.md:1]: @dgwatkins, I have another fix waiting for approval (https://github.com/OnTheGoSystems/WPML-MU-Domain-Mapping/pull/4). So I suggest that we bump the version later when the 2 fixes are merged. Could you please remove this?

File: plugin.php
Comment (strategio) [inline plugin.php:8]: Same here, please keep the current version number.

**Review by strategio — COMMENTED** [review]
@dgwatkins, it looks good to me. I didn't know these 2 plugins were sharing table names at this point. It looks like one is the fork of the other...

Concerning the version, I suggest to change it later to include the 2 fixes and then tag the branch.

**Comment by dgwatkins** [comment]
@strategio There it is, thanks

File: classes/class-wpml-mu-domain-mapping-requirements.php
Comment (andreasciamanna) [inline classes/class-wpml-mu-domain-mapping-requirements.php:37]: @dgwatkins as usual, the plugin folder might not be what we expect, even for must-use plugins.

What about checking if the `Domainmap_Plugin` class exists instead?

**Review by andreasciamanna — CHANGES_REQUESTED** [review]

File: classes/class-wpml-mu-domain-mapping-requirements.php
Comment (dgwatkins) [reply classes/class-wpml-mu-domain-mapping-requirements.php:37]: The reason is:

1) This plugin acts as a bridge for 2 different plugins
2) We check for the other one this way
3) So I checked this one in the same way

I can make the change to both though ..

Comment (dgwatkins) [reply classes/class-wpml-mu-domain-mapping-requirements.php:37]: @andreasciamanna Sorry, I don't think I should change anything

We are not just checking if the plugin is active in the current site, we are checking if its active in the network.

And I can't figure that out just by the existence of a class

File: changelog.md
Comment (dgwatkins) [reply changelog.md:1]: removed ages ago, but forgot to resolve the discussion

File: plugin.php
Comment (dgwatkins) [reply plugin.php:8]: same here

**Review by andreasciamanna — APPROVED** [review]

**Comment by dgwatkins** [comment]
What do you think @strategio ?
Are we ready to merge this and release ?
Let me know anything else you need from me

**Review by andreasciamanna — DISMISSED** [review]

**Comment by strategio** [comment]
Oh sorry @dgwatkins, I completely missed your request. It's fine to me! Maybe we should add a tag for this change. What do you think @andreasciamanna?

**Review by strategio — DISMISSED** [review]

**Comment by andreasciamanna** [comment]
@strategio What sort of tag? And where it should be added?

**Comment by strategio** [comment]
For the changelog @andreasciamanna https://github.com/OnTheGoSystems/WPML-MU-Domain-Mapping/blob/40e3fe61aad24b08a65d900ff759406d17ebb042/changelog.md.

**Comment by andreasciamanna** [comment]
@strategio you mean to release a new version, therefore **to tag **this project, not to "add a tag".

I thought you were talking about GitHub labels (aka tags):
image

Sure, this project is a rarely updated project: a new version is fine.

By the way, I don't know who's the maintainer or owner of this project, neither I know who usually publishes the updates.

**Comment by strategio** [comment]
@dgwatkins, I didn't notice that your PR comes from a fork. I wanted to remove your last "FIXUP" commit and reword your main commit, but I can't.

Could you please do this? (the version in the second commit is fine).

You can also reword the first commit like this:

>     Add support for domain-mapping plugin from WPMUDEV
> 
>     https://onthegosystems.myjetbrains.com/youtrack/issue/compsupp-2737

Then we can merge it.

**Comment by dgwatkins** [comment]
@strategio All set

**Review by strategio — APPROVED** [review]

**Comment by strategio** [comment]
@andreasciamanna, could you please add your approval again?

**Review by andreasciamanna — APPROVED** [review]

**Pull request merged**
